resources/js/components/ConvocatoriasComponent.vue

```diff
@@ -1,61 +1,59 @@
+                    <i class="fas fa-edit pin-r text-black text-xl-6 absolute invisible group-hover:visible"></i>
+                    <img class="w-full" src="https://tailwindcss.com/img/card-top.jpg" alt="">
+                    <div class="px-6 py-4">
+                        <div class="font-bold text-xl mb-2 text-grey-darkest">Título de la convocatoria</div>
+                        <p class="font-bold text-grey">Categoría</p> 
+                        <p class="my-3 text-grey-darker text-base">Descripción</p>
                     </div>
-                </a>
-            </div>
-            <div class="p-1 w-full sm:w-full md:w-1/2">
-                <a href="#" class="no-underline">
-                    <div class="group relative rounded overflow-hidden shadow-lg hover:opacity-75">
-                        <i class="fas fa-edit pin-r text-black text-xl-6 absolute invisible group-hover:visible"></i>
-                        <img class="w-full" src="https://tailwindcss.com/img/card-top.jpg" alt="">
-                        <div class="px-6 py-4">
-                            <div class="font-bold text-xl mb-2 text-grey-darkest">Título de la convocatoria</div>
-                            <p class="font-bold text-grey">Categoría</p> 
-                            <p class="my-3 text-grey-darker text-base">Descripción</p>
-                        </div>
-                        <div class="px-6 py-4">
-                            <span class="text-sm font-semibold text-grey-darker">Last updated 3 min ago</span>
-                        </div>
+                    <div class="px-6 py-4">
+                        <span class="text-sm font-semibold text-grey-darker">Last updated 3 min ago</span>
                     </div>
-                </a>
-            </div>
-            <div class="p-1 w-full sm:w-full md:w-1/2">
-                <a href="#" class="no-underline">
-                    <div class="group relative rounded overflow-hidden shadow-lg hover:opacity-75">
-                        <i class="fas fa-edit pin-r text-black text-xl-6 absolute invisible group-hover:visible"></i>
-                        <img class="w-full" src="https://tailwindcss.com/img/card-top.jpg" alt="">
-                        <div class="px-6 py-4">
-                            <div class="font-bold text-xl mb-2 text-grey-darkest">Título de la convocatoria</div>
-                            <p class="font-bold text-grey">Categoría</p> 
-                            <p class="my-3 text-grey-darker text-base">Descripción</p>
-                        </div>
-                        <div class="px-6 py-4">
-                            <span class="text-sm font-semibold text-grey-darker">Last updated 3 min ago</span>
-                        </div>
+                </div>
+            </router-link>
+        </div>
+        <div class="p-1 w-full sm:w-full md:w-1/2">
+            <router-link to="/convocatorias/edit" class="no-underline">
```

Comment:
For testing purposes consider send the `id` and in the logic of the component get the value usign props.

Example:

```javascript
// component
 export default {
        props: ['id'],

// routes
{ name: 'convocatoria_edit', path: '/convocatoria/editar/:id', component: ExComponent, props: true },

<router-link :to="{ name: 'convocatoria_edit', id:1 }"  class="no-underline">
```

Reply:
Si lo agrego como en el ejemplo que pusiste me dice que hace falta el campo id en la ruta, lo puse como data en el componente de inicio para que lo tomara de ahí, de todas formas despues cada convocatoria será un componente y su id estará en el data